pyproject.toml

```diff
@@ -17,11 +17,11 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = ">=3.8,<3.11"
```

Comment:
Just a note that once the next beta/rc of 2.0 is available, it'll include support for 3.11, so at that time this can become `<3.12`.

Reply:
The `<3.11` I understand is needed for b2 specifically, but once we're in rc1 is there any reason to cap the maximum version? The templates all use `^3.min` for python versions.

Reply:
The templates should be updated - core is being more defensive now by explicitly stating a maximum supported Python version instead of using `^3.min` as an implicit assertion that all newer Python versions will work. We have too many dependencies that do *not* necessarily support bleeding-edge Python versions for that to be a valid assertion.